### ISSUE TITLE
Security headers are missing

### ISSUE THREAD

**Issue by AkashThoriya** [bagisto/bagisto#5183]
# Bug Report
Few security-related headers are missing

## Issue Description
CSP policies headers are not here

<div class="reportSection">
<div class="reportTitle">Missing Headers</div>
<div class="reportBody">
<table class="reportTable">
<colgroup><col class="col1"><col class="col2"></colgroup>
<tbody>
<tr class="tableRow"><th class="tableLabel table_red">Content-Security-Policy</th><td class="tableCell"><a href="https://scotthelme.co.uk/content-security-policy-an-introduction/" target="_blank">Content Security Policy</a> is an effective measure to protect your site from XSS attacks. By whitelisting sources of approved content, you can prevent the browser from loading malicious assets.</td></tr><tr class="tableRow"><th class="tableLabel table_red">Permissions-Policy</th><td class="tableCell"><a href="https://scotthelme.co.uk/goodbye-feature-policy-and-hello-permissions-policy/" target="_blank">Permissions Policy</a> is a new header that allows a site to control which features and APIs can be used in the browser.</td></tr> </tbody>
</table>
</div>
</div>

## Bagisto Version
Latest version

## Steps To Reproduce
Generate demo via https://demo.bagisto.com/ and 
paste link into these sites
https://cspvalidator.org/
https://securityheaders.com/

## Actual Result
Headers are not there
![image](https://user-images.githubusercontent.com/53002296/135603370-0720f333-f120-47eb-8906-38a97709360f.png)
![image](https://user-images.githubusercontent.com/53002296/135603393-850ee66f-d608-43f3-be2a-7a64eaea003e.png)

## Expected Result
Headers must be there

**Comment by devansh-webkul** [comment]
Hi there,

Yes, the CSP header is missing because it impacts the whole content of the homepage, and I am also aware of that.

I will add this but needs to check the content also as this will block all the dynamic contents mostly.

**Comment by AkashWeybee** [comment]
> Hi @AkashThoriya,
> 
> Yes, the CSP header is missing because it impacts the whole content of the homepage, and I am also aware of that.
> 
> I will add this but needs to check the content also as this will block all the dynamic contents mostly.

Yes, you are right, it impacts the whole content of the homepage.
